gdx-pay-android-googlebilling/src/com/badlogic/gdx/pay/android/googlebilling/PurchaseManagerGoogleBilling.java

```diff
@@ -70,6 +82,14 @@ public void run() {
         });
     }
 
+    public void setObfuscatedAccountId(String obfuscatedAccountId) {
```

Comment:
Does it make sense to have two setter methods, or should these fields only set together?

Reply:
Google documentation doesn't make it clear. 

Since these are two separate fields in the builder, maybe we should leave them separate here as well.

What do you think?

Reply:
Yeah, that's fine